### PR TITLE
New working machine added

### PR DESCRIPTION
**Pull request by algestam**
----------
Teenage Mutant Ninja Turtles Basketball (Konami handheld) [algestam, Sean Riddle, Mr. Do]

### CODE REVIEW

File: src/mame/handheld/hh_sm510.cpp [hunk 1]
```diff
@@ -5608,6 +5608,70 @@ ROM_END
...
+}
+
+// roms
+
+ROM_START( ktmntbb )
+	ROM_REGION( 0x1000, "maincpu", 0 )
+	ROM_LOAD( "793.program", 0x0000, 0x1000, CRC(0973b329) SHA1(1c4161e5c53f6c6dd9752a228f361cf053f181f4) )
+
+	ROM_REGION( 0x100, "maincpu:melody", 0 )
+	ROM_LOAD( "793.melody", 0x000, 0x100, CRC(cddefb96) SHA1(e9b6d3947c415a3ed520a3dab66a381159dfd79b) )
```
Comment (cuavas) [inline src/mame/handheld/hh_sm510.cpp:5665]: Did this one actually have the melody ROM read from a decap?

Comment (algestam) [reply src/mame/handheld/hh_sm510.cpp:5665]: Yes, Sean has decapped the MCU and read out both the melody and program ROM.

Comment (cuavas) [reply src/mame/handheld/hh_sm510.cpp:5665]: Oh, cool.

**Comment by algestam** [comment]
Thanks @happppp , I'll check this.

**Comment by algestam** [comment]
Can confirm that the dangling leg part should be part of the ball-kick segment. Well spotted! Will prepare a fix in a separate PR.

**Pull request merged**